pkg/aws-s3/csv_test.go

```diff
@@ -132,7 +132,7 @@ func TestStreamingCSVToPage(t *testing.T) {
 	sampleHeaders := []string{"name", "age", "city", "aliases"}
 	attrConfigDefault := []*framework.AttributeConfig{
 		{ExternalId: "name", Type: framework.AttributeTypeString},
-		{ExternalId: "age", Type: framework.AttributeTypeInt64}, // Will be parsed as float64
+		{ExternalId: "age", Type: framework.AttributeTypeInt64}, // Will be parsed as int64
```

Comment:
nit: This can just be removed since this is now the default behavior
```suggestion
		{ExternalId: "age", Type: framework.AttributeTypeInt64},
```